components/org.wso2.carbon.secvault.securevault/src/main/java/org/wso2/carbon/secvault/securevault/SecureVaultUtils.java

```diff
@@ -49,13 +50,20 @@
 /**
  * Secure Vault utility methods.
  *
- * @since 5.2.0
+ * @since 5.0.0
  */
 public class SecureVaultUtils {
     private static final Logger logger = LoggerFactory.getLogger(SecureVaultUtils.class);
-    private static final String defaultCharset = StandardCharsets.UTF_8.name();
-    private static final Pattern varPatternEnv = Pattern.compile("\\$\\{env:([^}]*)}");
-    private static final Pattern varPatternSys = Pattern.compile("\\$\\{sys:([^}]*)}");
+    private static final String DEFAULT_CHARSET = StandardCharsets.UTF_8.name();
+    private static final Pattern VAR_PATTERN_ENV = Pattern.compile("\\$\\{env:([^}]*)}");
+    private static final Pattern VAR_PATTERN_SYS = Pattern.compile("\\$\\{sys:([^}]*)}");
```

Comment:
Yes my mistake Vidura, as Pattern.compile() and StandardCharsets.UTF_8.name() are immutable this is fine! Thanks!